backends/imgui_impl_wgpu.cpp

```diff
@@ -672,9 +672,15 @@ bool ImGui_ImplWGPU_CreateDeviceObjects()
     // Vertex input configuration
     WGPUVertexAttribute attribute_desc[] =
     {
+#ifndef __EMSCRIPTEN__
+        { {}, WGPUVertexFormat_Float32x2, (uint64_t)offsetof(ImDrawVert, pos), 0 },
+        { {}, WGPUVertexFormat_Float32x2, (uint64_t)offsetof(ImDrawVert, uv),  1 },
+        { {}, WGPUVertexFormat_Unorm8x4,  (uint64_t)offsetof(ImDrawVert, col), 2 },
+#else
         { WGPUVertexFormat_Float32x2, (uint64_t)offsetof(ImDrawVert, pos), 0 },
         { WGPUVertexFormat_Float32x2, (uint64_t)offsetof(ImDrawVert, uv),  1 },
         { WGPUVertexFormat_Unorm8x4,  (uint64_t)offsetof(ImDrawVert, col), 2 },
+#endif
```

Comment:
Is this fix compatible with the `VertexAttribute` struct in the Rust implementation? https://docs.rs/wgpu/latest/wgpu/struct.VertexAttribute.html

As far as I know, the `nextInChain` field at the front is exclusive to the Google Dawn implementation. I think `#ifdef IMGUI_IMPL_WEBGPU_BACKEND_DAWN` should be used here.